### PR TITLE
Fix minor problems found during going through the tutorial

### PR DESCRIPTION
**Pull request by aliok**
Went through the tutorial and found some minor problems.

### CODE REVIEW

File: documentation/modules/ROOT/pages/02-basic-fundas.adoc [hunk 1]
```diff
@@ -104,7 +104,7 @@ include::ROOT:partial$invoke-service.adoc[tag=*]
 :k8s-cli: oc
 include::ROOT:partial$invoke-service.adoc[tag=*]
 
-The last http command should return a response containing a line similar to **Hi  greeter => '6fee83923a9f' : 1**
```
Comment (aliok) [inline documentation/modules/ROOT/pages/02-basic-fundas.adoc:107]: The last command in that partial above is not the `http` command, so it was causing confusion.

File: documentation/modules/ROOT/pages/02-basic-fundas.adoc [hunk 2]
```diff
@@ -194,14 +194,12 @@ copyToClipboard::oc-basics-run-service-env[]
...
 After successful deployment of the service we should see a Kubernetes deployment called `greeter-cqgcf-deployment`.
 
-Now if you list revisions
+Now if you list revisions, you will see two of them, named similar to `greeter-cqgcf` and `greeter-nsrbr`.
 
 [#basics-show-knative-revisions]
 === revisions
 include::partial$knative-objects.adoc[tag=knative-revisions]
 
-, you will see two of them, named similar to `greeter-cqgcf` and `greeter-nsrbr`.
```
Comment (aliok) [inline documentation/modules/ROOT/pages/02-basic-fundas.adoc:203]: Moved this above instead of creating 2 parts for the sentence.

**Comment by aliok** [comment]
ping @kameshsampath

**Comment by kameshsampath** [comment]
thanks @aliok  - it should be in 0.5.0 release, which am planning to do next week

**Pull request merged**